src/db/migrations/20200110001514-create-user.js

```diff
@@ -28,16 +29,17 @@ module.exports = {
         allowNull: false
       },
       createdAt: {
-        allowNull: false,
-        type: Sequelize.DATE
+        type: Sequelize.DATE,
+        allowNull: false
+
```

Comment:
Please, remove this blank line. Use blank lines only between logic blocks. You can find an explanation for this online, but basically:

> Vertical indents: empty lines for splitting code into logical blocks.
> 
> Even a single function can often be divided into logical blocks. In the example below, the initialization of variables, the main loop and returning the result are split vertically:
> 
```
function pow(x, n) {
  let result = 1;
  //              <--
  for (let i = 0; i < n; i++) {
    result *= x;
  }
  //              <--
  return result;
```
> }
> Insert an extra newline where it helps to make the code more readable. There should not be more than nine lines of code without a vertical indentation.

Reply:
Yes, while I was changing the order to be more consistent I forgot to remove that. Thanks!